src/App.js

```diff
@@ -0,0 +1,64 @@
+import { useEffect } from "react";
+import { Route, Routes } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import "@ahaui/css/dist/index.min.css";
+import { getUserInfoAction, signOutAction } from "actions/user";
+import SignIn from "components/Auth/SignIn";
+import SignUp from "components/Auth/SignUp";
+import Home from "components/Home";
+import Toast from "components/Common/Toast";
+import CustomModal from "components/Common/Modal";
+import "./App.css";
+
+const App = () => {
+  // States
+  const isLoggedIn = useSelector((state) => state.user.isLoggedIn);
+  const modal = useSelector((state) => state.modal);
+  const dispatch = useDispatch();
+
+  // Auto sign in
+  useEffect(() => {
+    const autoSignIn = async (isLoggedIn) => {
+      if (isLoggedIn) {
+        const getUserInfoResult = await dispatch(getUserInfoAction());
+        if (!getUserInfoResult.success) {
+          if (String(getUserInfoResult.error.message).includes("Network Error"))
+            return;
+
+          dispatch(signOutAction());
+        }
+      }
+    };
+    autoSignIn(isLoggedIn);
```

Comment:
The purpose of the function is not for signing in automatically. Plz change to another name.

Reply:
Changed to keepUserSession, and use clearInfo action instead of signOut action